database/driver.mjs

```diff
@@ -116,6 +116,57 @@ class dbDriver {
     async find(query) {
         return this.controller.find(query).catch(err => err)
     }
+    async findOne(query) {
+        return this.controller.findOne(query).catch(err => err)
+    }
+
+    /**
+     * Get a database record by its ID.
+     * @param id The ID of the record to retrieve.
+     * @collection collection The collection or table to search.
+     * @return JSON of the matched document or standard error object
+     */
+    async getById(id, collection) {
+        return this.controller.getById(id, collection).catch(err => err)
+    }
+    /**
+     * 
+     * @param {*} payload body of project to be saved
+     * @param {*} collection collection of table
+     * @returns 
+     */
+
+    async saveProject(payload, collection) {
```

Comment:
I feel `saveProject()` should assume the Project collection.

Reply:
I can make it even easier on you and tell you that this function should be removed from this file.  It is neve used, and does what save() does.  saveProject() is used elsewhere in this codebase, but not through the driver.